lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -484,11 +486,11 @@ void _AddMissingTexcoordReaders(mx::DocumentPtr& mtlxDoc)
                             _mtlxTokens->geompropvalue.GetString(),
                             _mtlxTokens->ST_reader.GetString(),
                             _mtlxTokens->vector2.GetString());
-                        mx::ValueElementPtr prpInput
-                            = stReader->addInputFromNodeDef(_mtlxTokens->geomprop.GetString());
```

Comment:
addInputFromNodeDef will copy all properties from the NodeDef. We only need name and type.